src/ci/resources/papi_v2beta_application.conf

```diff
@@ -89,7 +89,7 @@ backend {
         include "papi_v2beta_provider_config.inc.conf"
         genomics.compute-service-account = "centaur@broad-dsde-cromwell-dev.iam.gserviceaccount.com"
         filesystems.http {}
-        docker-image-cache-manifest-file = "gs://cloud-cromwell-dev/docker-image-cache-manifest.json"
+        docker-image-cache-manifest-file = "gs://cloud-cromwell-dev/docker-image-cache-manifest-v2.json"
```

Comment:
TOL ISO 8601 embedded date formats might be nice, e.g. `docker-image-cache-manifest-2021-01-21.json`